.github/workflows/pipeline.yml

```diff
@@ -32,6 +32,15 @@ jobs:
             postgres: postgres
             workers: workers
 
+          - label: Py 3.10, PG 14, Monolith
+            sytest-tag: bookworm
```

Comment:
I think these refer to the tag on the `matrixdotorg/sytest-synapse` image, so should be `bookworm-python3.10` if we want python 3.10?